addGradientInfill.py

```diff
@@ -1,49 +1,55 @@
-##################################################
-## Gradient Infill for 3D prints
-##################################################
-## MIT license
-##################################################
-## Author: Stefan Hermann - CNC Kitchen
-## Version: 1.0
-##################################################
+#!/usr/bin/env python3
+"""
+Gradient Infill for 3D prints
+
+License: MIT
+Author: Stefan Hermann - CNC Kitchen
+Version: 1.0
+"""
 
 import re
-from collections import namedtuple
 from enum import Enum
+from collections import namedtuple
+
 
 class InfillType(Enum):
     SMALL_SEGMENTS = 1  # infill with small segments like honeycomb or gyroid
-    LINEAR = 2  # linear infill like rectilinear or triangles
+    LINEAR = 2          # linear infill like rectilinear or triangles
 
+    
 Point2D = namedtuple('Point2D', 'x y')
 Segment = namedtuple('Segment', 'point1 point2')
 
-################ EDIT this section for your creation parameters
+# EDIT this section for your creation parameters
 
 INPUT_FILE_NAME = "cloverleaf_wHole_gyroid.gcode"
 OUTPUT_FILE_NAME = "BOWDEN_cloverleaf_wHole_gyroid.gcode"
 
 infillType = InfillType.SMALL_SEGMENTS
 
-maxFlow = 350              # maximum extrusion flow
-minFlow = 50               # minimum extrusion flow
-gradientThickness = 6      # thickness of the gradient (max to min) in mm
-gradientDiscretization = 4 # only applicable for linear infills; number of segments within the 
-                           # gradient( segmentLength=gradientThickness/gradientDiscretization); 
-                           # use sensible values to not overload the printer
+maxFlow = 350               # maximum extrusion flow
+minFlow = 50                # minimum extrusion flow
+gradientThickness = 6       # thickness of the gradient (max to min) in mm
+gradientDiscretization = 4  # only applicable for linear infills; number of segments within the 
+                            # gradient( segmentLength=gradientThickness/gradientDiscretization);
+                            # use sensible values to not overload the printer
+
+# End edit
 
-###############################################################
 
 class Section(Enum):
     NOTHING = 0
     INNER_WALL = 1
     INFILL = 2
 
+
 def dist(segment, point):
-    # Calculate the distance from a point to a line with finite length
+    """
+    Calculate the distance from a point to a line with finite length
+    """
     px = segment.point2.x - segment.point1.x
     py = segment.point2.y - segment.point1.y
-    norm = px*px + py*py
+    norm = px * px + py * py
```

Comment:
Never forget the spaces around your operators or it can get really confusing. Like reading a 1,2 as a 1.2 etc.